### PR TITLE
pfe-199: Changed color for brand-seconday-hover, btn-active-color and…

### PR DESCRIPTION
**Pull request by Fastanton**
… btn-secondary-hover

## Description

the colors for brand-secondary-hover, --btn-active-color, --btn-secondary-hover was wrong. And need to change color. 

## Motivation and Context

The problems is solved with the corrections.

## How Has This Been Tested?

Visual review sent to designer.

## Types of changes

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [x] My code follows the code style of this project.
-   [ ] My change requires a change to the documentation.
-   [ ] I have updated the documentation accordingly.
-   [x] I have read the **CONTRIBUTING** document.
-   [ ] I have updated the **CHANGELOG** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.

## Review instructions

[Review instructions](../REVIEW_INSTRUCTIONS.md)

### CODE REVIEW

File: src/less/variables-payex.less [hunk 1]
```diff
@@ -157,7 +157,7 @@ body {
 	/* TODO: to be converted: --brand-primary-faded:  fade(var(--brand-primary, #2A9D3F), 60%); */
 
 	--brand-secondary: #3c3c3c;
-	--brand-secondary-hover: #5f5f5f;
+	--brand-secondary-hover: #707070;
```
Comment (goldenraphti) [inline src/less/variables-payex.less:160]: remember to keep "LESS vars" and "Pure CSS vars, i.e. custom properties" in sync 😉

File: src/less/components/payex/button.less [hunk 2]
```diff
@@ -33,7 +33,7 @@
 		}
 
 		&:active {
-			border: 1px solid @brand-secondary-info-1;
+			border: 1px solid @brand-primary;
```
Comment (goldenraphti) [inline src/less/components/payex/button.less:36]: remember to try to use CSS custom properties when possible and when it is not requiring too much work 😉

Comment (goldenraphti) [inline src/less/components/payex/button.less:36]:
```suggestion
			border: 1px solid var(--brand-primary);
```

**Review by goldenraphti — CHANGES_REQUESTED** [review]

File: src/less/variables-payex.less [hunk 3]
```diff
@@ -215,7 +215,7 @@ body {
 	--btn-secondary-bg: var(--brand-secondary, #3c3c3c);
 	--btn-secondary-text-color: var(--brand-secondary, #3c3c3c);
 	--btn-secondary-border-color: var(--brand-secondary, #3c3c3c);
-	--btn-secondary-hover: var(--brand-secondary, #3c3c3c);
+	--btn-secondary-hover: var(--soft-brown, #707070);
```
Comment (goldenraphti) [inline src/less/variables-payex.less:218]:
```suggestion
	--btn-secondary-hover: var(--brand-secondary-hover, #707070);
```

**Review by goldenraphti — APPROVED** [review]
Approooooved 💫✨💜

**Review by Joelhagg — APPROVED** [review]
Nicely done! If @goldenraphti give the thumbs up, so do I 👌

**Pull request merged**